### ISSUE TITLE
Popup screen not closing on Mobile device

### ISSUE THREAD

**Issue by samgithub-duplicate** [MrSwitch/hello.js#218]
Hello.js works fine when run from the desktop OR using the chrome browser to navigate to the site in Android device.  

When I "add it to the homescreen" and try it run it in "standalone" mode (ie. without a visible address bar) I get the following results

a) Singin with google (gmail)   -- WORKS
b) Hotmail/Outlook   --  page get's stuck of redirect page. It does not CLOSE.  It has to be closed MANUALY by the user.  Then it works.
c) Facebook  -- same problem as for (b)

Basically, it appears the popup( consent/redirect) screen overlays the signing page and does not close as in (a)

**Comment by MrSwitch** [comment]
So as long as its not running as a Google Chrome App it should still work.

**Comment by MrSwitch** [comment]
inactive
